src/System.ObjectModel/tests/ObservableCollection/ObservableCollection_ConstructorAndPropertyTests.netstandard1.7.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Xunit;
+
+namespace System.Collections.ObjectModel.Tests
+{
+    /// <summary>
+    /// Tests the public properties and constructor in ObservableCollection<T>.
+    /// </summary>
+    public partial class ConstructorAndPropertyTests
+    {
+        /// <summary>
+        /// Tests that ArgumentNullException is thrown when given a null IEnumerable.
+        /// </summary>
+        [Fact]
+        public static void IEnumerableConstructorTest_Negative()
+        {
+            Assert.Throws<ArgumentNullException>("list", () => new ObservableCollection<string>(null));
+        }
+
+        [Fact]
+        public static void IListConstructorTest()
+        {
+            List<string> collection = new List<string> { "one", "two", "three" };
+            var actual = new ObservableCollection<string>((List<string>)collection);
```

Comment:
The cast shouldn't be necessary, since it is type as `List<string>` it will use the right ctor, e.g.:

```c#
var actual = new ObservableCollection<string>(collection);
```